needle-slack-connector/src/server/db/schema.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  pgTableCreator,
+  serial,
+  text,
+  timestamp,
+  varchar,
+  jsonb,
+} from "drizzle-orm/pg-core";
+
+export const createTable = pgTableCreator((name) => name);
+
+export interface FileMetadata {
+  channelId: string;
+  monthStart: string;
+  monthEnd: string;
+  dataType: string;
+}
+
+export interface CanvasFileMetadata {
+  channelId: string;
+  originId: string;
+  url: string;
+  title: string;
+  dataType: string;
+}
+
+export const filesTable = createTable("files", {
+  id: serial("id").primaryKey(),
+  ndlConnectorId: varchar("ndl_connector_id", { length: 256 }).notNull(),
+  ndlFileId: varchar("ndl_file_id", { length: 256 }).notNull(),
+  title: varchar("title", { length: 512 }),
+  metadata: jsonb("metadata")
```

Comment:
why do we use `jsonb` instead of spreading each field as its' own column? i find latter is more idiomatic psql.

Reply:
I spreaded the fields! Looking better now 👍